Dockerfile.prow

```diff
@@ -1,5 +1,12 @@
 # Copyright (c) 2021 Red Hat, Inc.
 # Copyright Contributors to the Open Cluster Management project.
+FROM registry.ci.openshift.org/open-cluster-management/builder:go1.15-linux-amd64 AS builder
```

Comment:
Do not use this image. In order to support s390x, the image must support multi-arch. so replace it with `xxx/builder:go1.15-linux`

Reply:
done